resources/startup.sh

```diff
@@ -142,9 +148,18 @@ if [ "$(doguctl config successfulInitialConfiguration)" != "true" ]; then
   exportNexusPassword
 
   export NEXUS_SERVER="http://localhost:8081/nexus"
-  nexus-claim plan -i /defaultDockerRegistry.hcl -o "-" | nexus-claim apply -i "-"
+
+  # Install default docker registry if requested via etcd
+  if doguctl config installDefaultDockerRegistry > /dev/null ; then
```

Comment:
Is the double-if really necessary? Maybe we could work with one statement, an & or the --default parameter of doguctl.